install.py

```diff
@@ -55,6 +63,8 @@ def linux_installation():
     if path == "":
         path = default_path
 
+    path = expand_tilde(path)
```

Comment:
It looks like you can just call expanduser(path) here, and if ~ is at the beginning it will expand it, no need for the function.